### PR TITLE
Add fmriprep-fake from djarecka -- useful for trying/testing pipelines etc

### PR DESCRIPTION
**Pull request by yarikoptic**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -447,3 +447,12 @@ apps:
   - gh: "Medical-Image-Analysis-Laboratory/mialsuperresolutiontoolkit"
     status: active
     dh: "sebastientourbier/mialsuperresolutiontoolkit"
+
+  - gh: "djarecka/fmriprep-fake"
+    status: active
+    dh: "djarecka/fmriprep_fake"
+    ds_type:
```
Comment (Remi-Gau) [inline _config.yml:454]: can specify that there is no CI for this app: 

see template here

https://github.com/bids-apps/bids-apps.github.io/blob/89c84714819c787d2e7245d478693f7b6c190c25/_config.yml#L78

File: _config.yml [hunk 2]
```diff
@@ -447,3 +447,12 @@ apps:
   - gh: "Medical-Image-Analysis-Laboratory/mialsuperresolutiontoolkit"
     status: active
     dh: "sebastientourbier/mialsuperresolutiontoolkit"
+
+  - gh: "djarecka/fmriprep-fake"
+    status: active
+    dh: "djarecka/fmriprep_fake"
```
Comment (Remi-Gau) [inline _config.yml:453]: FWI: apps that are not on the bids app orga are sorted alphabetically based on the `dh` key

https://github.com/bids-apps/bids-apps.github.io/blob/89c84714819c787d2e7245d478693f7b6c190c25/_config.yml#L300-L301

**Comment by Remi-Gau** [comment]
had a super quick look at the readme of the app:

I suspect the `=` in `--participant-label=sub-1` for the CLI is a typo, right?

**Comment by djarecka** [comment]
> had a super quick look at the readme of the app:
> 
> I suspect the `=` in `--participant-label=sub-1` for the CLI is a typo, right?

it should work fine with "="

**Pull request merged**